src/openforms/forms/tests/e2e_tests/test_logic_tab.py

```diff
@@ -86,3 +86,79 @@ def assertState():
                 )
 
             await assertState()
+
+    @skip_on_webtest
+    async def test_evaluate_dmn_missing_configuration_in_logic(self):
+        @sync_to_async
+        def setUpTestData():
+            # set up a form
+            form = FormFactory.create(
+                name="Form with logic",
+                formstep__form_definition__slug="form-with-logic",
+                formstep__form_definition__name="Some step",
+                formstep__form_definition__configuration={
+                    "components": [
+                        {
+                            "type": "textfield",
+                            "key": "field1",
+                            "label": "Some Field",
+                        },
+                    ],
+                },
+                generate_minimal_setup=True,
+            )
+            return form
+
+        await create_superuser()
+        form = await setUpTestData()
+        admin_url = str(
+            furl(self.live_server_url)
+            / reverse("admin:forms_form_change", args=(form.pk,))
+        )
+
+        async with browser_page() as page:
+            # Goto "Logic"
+            await self._admin_login(page)
+            await page.goto(str(admin_url))
+            await page.get_by_role("tab", name="Logic").click()
+            await page.get_by_text("Add rule").click()
+            await page.get_by_role("button", name="Advanced").click()
+
+            await enter_json_in_editor(page, page.locator(".monaco-editor"), True)
+
+            # Add 'Evaluate DMN' action without config
+            await page.get_by_text("Add Action").click()
+            await page.locator('select[name="action.type"]').select_option(
+                label="Evaluate DMN"
+            )
+            await expect(
+                page.get_by_text("DMN configuration:(not configured yet)")
+            ).to_be_visible()
+
+            await page.get_by_text("Save and continue editing").click()
+
+            change_form_header = page.get_by_role("heading", name="Change form")
+
+            await expect(change_form_header).to_be_visible()
+
+            async def assertState():
+                logic_tab = page.get_by_role("tab", name="Logic")
+
+                await expect(
+                    logic_tab.get_by_title("There are validation errors")
+                ).to_be_visible()
+
+                await logic_tab.click()
+
+                # Logic shows error for DMN config
+                node_with_error = page.locator("css=.dsl-editor__node--errors")
+                await expect(
+                    node_with_error.get_by_text("This field is required")
+                ).to_be_visible()
+                await expect(
+                    node_with_error.get_by_text(
+                        "DMN configuration:(not configured yet)"
+                    )
+                ).to_be_visible()
+
+            await assertState()
```

Comment:
```suggestion

            logic_tab = page.get_by_role("tab", name="Logic")

            await expect(
                logic_tab.get_by_title("There are validation errors")
            ).to_be_visible()

            await logic_tab.click()

            # Logic shows error for DMN config
            node_with_error = page.locator("css=.dsl-editor__node--errors")
            await expect(
                node_with_error.get_by_text("This field is required")
            ).to_be_visible()
            await expect(
                node_with_error.get_by_text(
                    "DMN configuration:(not configured yet)"
                )
            ).to_be_visible()
```
the `assertState` workaround is to be able to make DB queries in an async context - if you're not doing that, you can just inline the rest of the test code

Reply:
Check. Yeah if that storybook test approach is a bust, i'll remove that `assertState` function call

Reply:
Update: I've removed this e2e test and added a storybook test which validates that DMN configuration errors are shown in the logic view.